spec/outputs/elasticsearch_spec.rb

```diff
@@ -597,6 +612,12 @@ def mock_actions_with_response(*resp)
         end
 
         it "successful requests should not be appended to retry queue" do
+          # insist indices are empty
+          Stud::try(10.times) do
+            r = @es.search
+            insist { r["hits"]["total"] } == 0
```

Comment:
Why not moving out of insist as we did in core? we can do most of this expectations by simply using the expect rspec method.